test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerTest.java

```diff
@@ -79,96 +76,50 @@ public void before() {
     public void after() {
         helper.deleteUser(testUser);
     }
-
+    
     @Test
-    public void canSignOut() {
+    public void canSignIn() {
         running(testServer(3333), new TestUtils.FailableRunnable() {
             public void testCode() throws Exception {
                 ObjectNode node = JsonNodeFactory.instance.objectNode();
-                node.put(STUDY_PROPERTY, testUser.getStudyIdentifier().getIdentifier());
+                node.put(STUDY_PROPERTY, testUser.getStudy().getIdentifier());
                 node.put(USERNAME, testUser.getUsername());
                 node.put(PASSWORD, testUser.getPassword());
                 
                 WSRequest request = WS.url(TEST_BASE_URL + SIGN_IN_URL);
                 WSResponse response = request.post(node).get(TIMEOUT);
+                
+                String sessionToken = new ObjectMapper().readTree(response.getBody()).get("sessionToken").asText();
 
-                WSCookie cookie = response.getCookie(BridgeConstants.SESSION_TOKEN_HEADER);
-
-                // All of a sudden, there's no cookie being set. I have no idea why.
-                assertNotNull("There's a cookie", cookie);
-
-                String sessionToken = cookie.getValue();
-                assertTrue("Cookie is not empty", StringUtils.isNotBlank(sessionToken));
-
-                ObjectNode emptyNode = JsonNodeFactory.instance.objectNode();
-                response = WS.url(TEST_BASE_URL + SIGN_OUT_URL)
-                        .setHeader(BridgeConstants.SESSION_TOKEN_HEADER, cookie.getValue()).post(emptyNode).get(TIMEOUT);
-
-                cookie = response.getCookie(BridgeConstants.SESSION_TOKEN_HEADER);
-                assertEquals("Cookie has been set to empty string", "", cookie.getValue());
-
-                String output = jedisOps.get(sessionToken);
-                assertNull("Should no longer be session data", output);
+                // There's actually no way to *request* stuff from another study at this point,
+                // so this test hardly makes sense at this point.
+                request = WS.url(TEST_BASE_URL + SCHEDULES_API);
```

Comment:
Is this typo that we're testing against the schedules API?  In the meantime, the request/response in the above lines are about sign-in.  But they are not tested and get overwritten by the tests here.  Should we just test the sign-in here?